build.gradle

```diff
@@ -11,19 +11,14 @@ plugins {
     id 'signing'
     id 'jacoco'
     id 'com.palantir.git-version' version '0.5.1'
```

Comment:
Cannot you update also the git-version?

Reply:
That's a good point.  I had avoided it because I didn't want any version name surprises, but it's worthing looking at.

Reply:
It seems fine, I've updated it.